Doc/library/plistlib.rst

```diff
@@ -172,13 +175,20 @@ Generating a plist::
         ),
         someData = b"<binary gunk>",
         someMoreData = b"<lots of binary gunk>" * 10,
-        aDate = datetime.datetime.fromtimestamp(time.mktime(time.gmtime())),
+        aDate = datetime.datetime.now()
     )
-    with open(fileName, 'wb') as fp:
-        dump(pl, fp)
+    with open("blah.plist", 'wb') as fp:
+        plistlib.dump(pl, fp)
```

Comment:
```suggestion
    print(plistlib.dumps(pl).decode())
```
I checked that example runs.  With this change, it is easy to see what is produced.